src/blocks/popup.css

```diff
@@ -19,275 +19,204 @@
   overflow: hidden;
   visibility: visible;
   opacity: 1;
+  transition: visibility 0s, opacity 0.6s;
 }
 
 .popup__container {
   position: relative;
-  top: 57px;
-  left: 0px;
-  background-color: #ffffff;
+  background-color: white;
   width: 100%;
-  max-width: 430px;
-  min-height: 331px;
-  height: auto;
-  border: 1px solid #000;
+  max-width: 358px;
+  max-height: 342px;
```

Comment:
Please, delete `max-height: 342px;` from `.popup__container` because it hardcodes the height which is not needed for the `confirmation` modal